### DISCUSSION TITLE
提一个小问题

### DISCUSSION THREAD

**Discussion by theblueandblackcrow** [the1812/Bilibili-Evolved#2803] [功能建议 / Ideas]
动态和历史记录什么时候能够像稍后再看那样不用网页f5就能刷新呢？
以前用的v1版本是能够做到的，现在好像没有适配好

**Comment by Geeyun-JY3** [comment]
动态见：#2737

**Comment by the1812** [comment]
历史记录 v2 支持无限滚动了, 所以和收藏类似, 不能再自动刷新了. 不过我觉得可以放个手动刷新的按钮在上面

**Reply by theblueandblackcrow** [reply]
emmmmm……
可能我的表述有误哈
我的这个意思是
动态和历史的刷新指的是在一段时间内更新新的内容
例子：比如某某up发布一个视频，你往上翻就能看到这个，历史记录也同理
现在v2版本好像除了稍后再看能够做到这一点，动态、历史记录好像都不行，还是最新beta版🤔

**Reply by the1812** [reply]
历史记录 v1 是在你打开的一瞬间刷新的, 动态才是定时刷新, 定时刷新 v2 里还没弄完

**Reply by Geeyun-JY3** [reply]
v2 历史记录的这个无限滚动，是 #927 吗？
